compiler/src/dmd/statementsem.d

```diff
@@ -2758,6 +2758,12 @@ Statement statementSemanticVisit(Statement s, Scope* sc)
                 {
                     if (e0)
                         rs.error("expected return type of `%s`, not `%s`", tret.toChars(), resType.toChars());
+                    else if (tbret.isTypeNoreturn())
+                    {
+                        rs.error("cannot return from `noreturn` function");
+                        .errorSupplemental(rs.loc,
+                            "perhaps add an endless loop, `assert(0)`, or another `noreturn` expression");
```

Comment:
```suggestion
                            "Consider adding an endless loop, `assert(0)`, or another `noreturn` expression");
```

Reply:
errors are not capitalized though, and there are plenty more "perhaps do ..." messages while there's only 1 "Consider doing ..." message

Reply:
"perhaps" seems vague and meandering whereas "consider" is to the point. Even better would be to scrap the sentence structure it as more of an "FYI You can do XYZ" but at that point it gets quite long. Ideally we'd have error codes you can read about on dlang.org but these were vetoed when I mentioned it in a meeting.

Capitalize as you prefer although the supplemental is obviously a new sentence.